ajax/libs/noti.js/package.json

```diff
@@ -16,6 +16,6 @@
   "license": "GPL-2.0",
   "repository": {
     "type": "git",
-    "url": "https://github.com/j-l-n/noti.js.git"
+    "url": "https://github.com/needim/noty.git"
```

Comment:
btw, @hare1039 please also help confirm the change, you guys can have a discussion first, thanks.